icons/spell-check.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m6 16 6-12 6 12" />
+  <path d="M8 12h8" />
+  <path d="m16 20 2 2 4-4" />
+</svg>
```

Comment:
If you want to give credit to someone, you can add them as a co-author.

`Co-authored-by: Jakob Guddas <github@jguddas.de`

https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reply:
Will do!